wordcount/data.go

```diff
@@ -99,10 +101,28 @@ func splitData(fileName string, chunkSize int) (numMapFiles int, err error) {
 	//
 	// 	Use the mapFileName function generate the name of the files!
 
-	/////////////////////////
-	// YOUR CODE GOES HERE //
-	/////////////////////////
-	numMapFiles = 0
```

Comment:
Foi só bisonhice minha mesmo. 

Fiz um Split bem força bruta (sem considerar possibilidade de cortar a palavra), meus testes estão passando, mas sei que não considerei esse caso. Até tentei implementar (deixei o código comentado), mas estava ficando enorme e não conseguia testar nada...

A ideia era ir lendo o texto movendo "chunck" bytes e então regredir caractere por caractere até achar um separador "X".

Cada partição do texto seria então:

##############(Separador)###(Chunk)######################
##############(Parte 1)#################################
##############(Parte 1)#################(Separador)##(Chunk)
##############(Parte 1)#################(Parte 2)##########
##############(Parte 1)#################(Parte 2)#####(Parte 3)